ziti/cmd/ops/verify/ext-jwt-signer/oidc/oidc.go

```diff
@@ -402,6 +408,30 @@ func NewOidcVerificationCmd(out io.Writer, errOut io.Writer, initialContext cont
 	return cmd
 }
 
+/* jwtPayload extracts the payload from a jwt so the contents can be logged shown to the user
+ */
+func jwtPayload(token string) (string, error) {
+	parts := strings.Split(token, ".")
+	if len(token) < 2 {
```

Comment:
Shouldn't this be `len(token) == 3` ?

This will not detect a token that has  3 or more periods in it.